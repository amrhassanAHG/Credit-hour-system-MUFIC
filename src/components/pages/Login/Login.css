.login-page{
    background-image: url('../../../assets/1.jpg');
}

.login-logo a b{
    margin-right: 12px;
    font-size: 30px;
}