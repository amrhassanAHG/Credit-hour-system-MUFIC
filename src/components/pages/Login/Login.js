import React, { Component } from "react";
import { NavLink } from "react-router-dom";
import './Login.css'

export default class Page extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div className="login-page">
        <div className="login-box">
          <div className="login-logo">
            <a href="/">
              <b style={{ color: "white" }}>MUFCI</b>
              <b style={{ color: "white" }}>System</b>
            </a>
          </div>
          {/* /.login-logo */}
          <div className="card">
            <div className="card-body login-card-body">
              <p className="login-box-msg">Sign in to start your session</p>
              <form action="../../index3.html" method="post">
                <div className="input-group mb-3">
                  <input
                    type="email"
                    className="form-control"
                    placeholder="Email"
                  />
                  <div className="input-group-append">
                    <div className="input-group-text">
                      <span className="fas fa-envelope" />
                    </div>
                  </div>
                </div>
                <div className="input-group mb-3">
                  <input
                    type="password"
                    className="form-control"
                    placeholder="Password"
                  />
                  <div className="input-group-append">
                    <div className="input-group-text">
                      <span className="fas fa-lock" />
                    </div>
                  </div>
                </div>
                <div className="row">
                  <div className="col-8">
                    <div className="icheck-primary">
                      <input type="checkbox" id="remember" />
                      <label htmlFor="remember">Remember Me</label>
                    </div>
                  </div>
                  {/* /.col */}
                  <div className="col-4">
                    <button type="submit" className="btn btn-primary btn-block">
                      Sign In
                    </button>
                  </div>
                  {/* /.col */}
                </div>
              </form>
              <p className="mb-1">
                <a href="/forget-password">I forgot my password</a>
              </p>
            </div>
            {/* /.login-card-body */}
          </div>
        </div>
      </div>
    );
  }
}
