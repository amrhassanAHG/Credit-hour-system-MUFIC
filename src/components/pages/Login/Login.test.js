import React from 'react';
import renderer from 'react-test-renderer';
import Login from './Login';

test('Login Testing', () => {
    const component = renderer.create(
      <Login ></Login>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
