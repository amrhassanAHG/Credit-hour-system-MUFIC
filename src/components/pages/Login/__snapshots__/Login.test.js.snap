// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Testing 1`] = `
<div
  className="login-page"
>
  <div
    className="login-box"
  >
    <div
      className="login-logo"
    >
      <a
        href="/"
      >
        <b
          style={
            Object {
              "color": "white",
            }
          }
        >
          MUFCI
        </b>
        <b
          style={
            Object {
              "color": "white",
            }
          }
        >
          System
        </b>
      </a>
    </div>
    <div
      className="card"
    >
      <div
        className="card-body login-card-body"
      >
        <p
          className="login-box-msg"
        >
          Sign in to start your session
        </p>
        <form
          action="../../index3.html"
          onSubmit={[Function]}
        >
          <div
            className="input-group mb-3"
          >
            <input
              className="form-control"
              id="login-email"
              placeholder="Email or username"
              type="text"
            />
            <div
              className="input-group-append"
            >
              <div
                className="input-group-text"
              >
                <span
                  className="fas fa-envelope"
                />
              </div>
            </div>
          </div>
          <div
            className="input-group mb-3"
          >
            <input
              className="form-control"
              id="login-password"
              placeholder="Password"
              type="password"
            />
            <div
              className="input-group-append"
            >
              <div
                className="input-group-text"
              >
                <span
                  className="fas fa-lock"
                />
              </div>
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="col-8"
            >
              <div
                className="icheck-primary"
              >
                <input
                  id="remember-me"
                  type="checkbox"
                />
                <label
                  htmlFor="remember-me"
                >
                  Remember Me
                </label>
              </div>
            </div>
            <div
              className="col-4"
            >
              <button
                className="btn btn-primary btn-block"
                type="submit"
              >
                Sign In
              </button>
            </div>
          </div>
        </form>
        <div
          id="error-message"
          style={
            Object {
              "color": "red",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
